### PR TITLE
[DEV-5135] Add NIA fields to api/v/awards/<award-id> response

### PR DESCRIPTION
**Pull request by tony-sappe**
**Description:**
Adding pre-existing FPDS fields to procurement awards' response

**Technical details:**
N/A

**Requirements for PR merge:**

1. [x] Unit & integration tests updated
2. [x] API documentation updated
3. [x] Necessary PR reviewers:
    - [x] Backend
    - [x] Frontend
4. [x] Matview impact assessment completed (N/A)
5. [x] Frontend impact assessment completed (N/A)
6. [x] Data validation completed
7. [x] Appropriate Operations ticket(s) created (N/A)
8. [x] Jira Ticket [DEV-5135](https://federal-spending-transparency.atlassian.net/browse/DEV-5135):
    - [x] Link to this Pull-Request
    - [x] Performance evaluation of affected API (No change)
    - [x] Before / After data comparison

**Area for explaining above N/A when needed:**
```
Simple API response change, No impact to database
```

### CODE REVIEW

File: usaspending_api/api_contracts/contracts/v2/awards/award_id.md [hunk 1]
```diff
@@ -77,6 +77,7 @@ This endpoint returns a list of data that is associated with the award profile p
                     "multi_year_contract": "N",
                     "consolidated_contract": "N",
                     "type_of_contract_pricing": "J",
+                    "national_interest_action": "NONE",
```
Comment (sethstoudenmier) [inline usaspending_api/api_contracts/contracts/v2/awards/award_id.md:80]: Is there any reason this isn't paired with the `national_interest_action_description` below?

Comment (tony-sappe) [reply usaspending_api/api_contracts/contracts/v2/awards/award_id.md:80]: I copied the actual API JSON response into the markdown

File: usaspending_api/api_contracts/contracts/v2/awards/award_id.md [hunk 2]
```diff
@@ -667,6 +677,8 @@ This endpoint returns a list of data that is associated with the award profile p
     `null` if the contract has no parent
 + `naics_hierarchy` (required, NAICSHierarchy, fixed-type)
 + `psc_hierarchy` (required, PSCHierarchy, fixed-type)
++ `national_interest_action` (required, string)
```
Comment (ebdabbs) [inline usaspending_api/api_contracts/contracts/v2/awards/award_id.md:680]: Should these also be `nullable`?

Comment (tony-sappe) [reply usaspending_api/api_contracts/contracts/v2/awards/award_id.md:680]: Yes, unsure how I missed that

File: usaspending_api/api_contracts/contracts/v2/awards/award_id.md (hunk 1 above)
Comment (tony-sappe) [reply usaspending_api/api_contracts/contracts/v2/awards/award_id.md:80]: Moved

File: usaspending_api/api_contracts/contracts/v2/awards/award_id.md (hunk 2 above)
Comment (tony-sappe) [reply usaspending_api/api_contracts/contracts/v2/awards/award_id.md]: Ha, they were defined in the wrong spot too. Corrected both issues

Comment (ebdabbs) [reply usaspending_api/api_contracts/contracts/v2/awards/award_id.md]: TY!

**Pull request merged**